CONTRIBUTING.md

```diff
@@ -113,3 +114,54 @@ that value later on.
 * Many relationships change between methods prefixed with `get` like
 `getProject()` and virtuals like `project`. Make sure you keep track of what you
 want to call.
+
+## Aptly
+
+While Houston itself does not host any repositories, it does tie into them to
+ensure released projects are available. [Aptly](http://www.aptly.info/) is the
+repository manager of choice for Houston and as such, it is important to have
+an able and working setup if you plan on utilizing the full power of Houston.
+
+### Setup
+
+This guide will not discuss installing Aptly. Please direct your attention to
+the [Aptly download page](http://www.aptly.info/download/) for details on
+installing and running.
+
+Houston's repository schema, while flexible, does make assumptions on your
+setup. First, we expect you to prefix all your repositories with distributions.
+For example, if you are having Houston build for Debian Sid, your review
+repository is called 'review', and your stable repository is called 'houston',
+then your full repository name should be 'sid-testing' and 'sid-houston'. By
```

Comment:
hm isn't testing kind of more traditionally the name here? Maybe "houston" isn't as descript as it could be though. Could be "release" or "production" or "stable" or something instead.